### PR TITLE
Feature 507: Implement usage of aliases in grafana dashboard

### LINKED ISSUE #507: Implement View of Aliases in Grafana Dashboard
## User stories
As a user I want to see aliases for config_id and run_id in the dashboard in order to get an easier overview.

## Acceptance Criteria
- [x] adapt all grafana functions to use aliases
- [x] adapt all datascience functions to use aliases

### PR DESCRIPTION
**Pull request by LucasDerReisende**
Fixes #507

## PR checklist

- [x] Acceptance criteria fulfilled
- [x] Additional features are tested
- [x] Docs (code, wiki & diagrams) updated
- [x] Breaking changes anounced
- [x] Added classes that inherit from `BaseModel` to `src.constants.tables`
- [x] Dev-branch has been merged into local branch to resolve conflicts
- [x] Tests and linter have passed AFTER local merge
- [ ] Another dev reviewed and approved

### CODE REVIEW

File: tests/data_science/test_grafana_data_registration.py [hunk 1]
```diff
@@ -49,15 +49,15 @@ def setup_method(self):
...
 
     @pytest.fixture
     def _config_id(self, simulation_configuration):
-        return str(simulation_configuration.id)
+        return str(simulation_configuration.readable_id)
 
     @pytest.fixture
     def _multi_config(self, simulation_configuration):
-        return str(simulation_configuration)
+        return str(simulation_configuration.readable_id)
```
Comment (instant-sky) [inline tests/data_science/test_grafana_data_registration.py:61]: This didn't use the id previously, should it now use the readable id?

**Review by instant-sky — COMMENTED** [review]
Nice, that should improve the readability in the dashboard by a lot! 📈

File: tests/data_science/test_grafana_data_registration.py (hunk 1 above)
Comment (LucasDerReisende) [reply tests/data_science/test_grafana_data_registration.py:61]: When calling str() on simulation_configuration, the ID will automatically be used to generate the str, I believe. So when we are now switching to readable ID, I have to specify that directly.

Comment (instant-sky) [reply tests/data_science/test_grafana_data_registration.py:61]: Ah okay, thank you for the clarification!

**Review by instant-sky — APPROVED** [review]

**Pull request merged**